tests/automation_framework/data/worker/workerregister_organizationId_empty.json

```diff
@@ -0,0 +1,25 @@
+{
+    "jsonrpc": "2.0",
+    "method": "WorkerRegister",
+    "id": 1,
+    "params": {
+        "workerId":"",
+        "workerType": 1,
+        "organizationId":" ",
```

Comment:
Here "organizationId":" " is not empty but I see a space between opening and closing double quotes.